promgen/templates/promgen/project_detail.html

```diff
@@ -29,23 +29,23 @@ <h1>
 
 {% include "promgen/project_detail_configuration.html" %}
 
-<div class="panel panel-danger" v-cloak v-if="alertLabelsProject.has('{{project.name}}')">
+<div class="panel panel-danger" v-cloak v-if="activeProjectAlerts.has('{{project.name}}')">
   <div class="panel-heading">
     <a @click="toggleCollapse('alerts-project-{{project.name|slugify}}')" class="btn btn-danger btn-sm" role="button">Alerts</a>
   </div>
   <table id="alerts-project-{{project.name|slugify}}" class="table table-bordered table-condensed">
-    <tr v-for="(alert, index) in globalAlerts" v-if="alert.labels.project == '{{project.name}}'">
+    <tr v-for="alert in activeProjectAlerts.get('{{project.name}}')">
```

Comment:
The behavior changed here (ditto for other places in this PR). Since the previous code was iterating over `globalAlerts`, it would also display inactive (suppressed) alerts, but only if there's any other active alerts (due to `alertLabelsProject.has()`).

The new logic only displays active alerts now. Is this change acceptable?

Reply:
Good catch on the bug. The global alerts should probably only show alerts set to active (not silenced) though the project/service specific ones should probably show all of them, with an extra indicator if it is silenced or not.

Reply:
I think this will work as is, and we can think about if it needs to be changed later 😄

Reply:
Sure, I can work on this after I finish with my currently planned PRs